packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -1099,6 +1107,15 @@ abstract class ResidentRunner {
     );
   }
 
+  void printStructuredErrorLog(vm_service.Event event) {
+    if (event.extensionKind == 'Flutter.Error') {
+      final Map<dynamic, dynamic> json = event.extensionData?.data;
+      if (json != null && json.containsKey('renderedErrorText')) {
+        globals.printStatus('\n${json['renderedErrorText']}');
```

Comment:
the existing output prefixes each line in the error with `flutter: `
@jonahwilliams will this replicate that behavior and is that desirable?

Reply:
I thought that was only for logs that go through logcat

Reply:
The existing code prefixes with `flutter: ` for my test of throwing an early error, but the new code does not prefix with `flutter: `. I figured since this is similar to the change in the flutter-intellij plugin (using structured errors there will stop prefixing) it would make sense to have the change here but can adjust if the prefix is useful.

Reply:
Without the prefix is fine